### ISSUE TITLE
Misleading Service Restart Requests - Leading to user potential confusion PF11.1

### ISSUE THREAD

**Issue by PureOxygen-Dev** [inverse-inc/packetfence#6818]
**Describe the bug**
At various locations within the GUI the system when saving offers advice on which service needs to be restarted.  However, within the platform the GUI at times presents services that need to be restarted that are disabled.  This leads to confusion to the user when they select the restart option as it presents an error message within the top right notification bubbles.   

I have included one example of this with screenshots but have seen the experience in other locations of the GUI too.

**To Reproduce**
1. Go to Status | Services
2. Scroll down the 'Manageable Services'
3. Your configuration needs to have 'radiusd-acct' disabled but running

![image](https://user-images.githubusercontent.com/95610834/149083742-78729b4f-24eb-4054-9ef9-2ee8cbdfcfb6.png)

4. Go to Configuration | System Configuration | Radius | EAP Profiles.
5. Click on default 
6. Click save and you will see the following two services need restarting.

![image](https://user-images.githubusercontent.com/95610834/149083878-e10b1d67-96df-4b08-9a9f-9d350daf4c05.png)

7. Click in the dropdown for radiusd-acct and choose restart
8. You will see the following error appear

![image](https://user-images.githubusercontent.com/95610834/149084080-9aed8129-223a-48e3-8514-387a132ac941.png)

9. Click in the dropdown for radiusd-auth and choose restart
10. You will see the following success message appear (The service text is also in red, and it that dictates error or stop.  The text might be better positioned in green meaning go or success).

![image](https://user-images.githubusercontent.com/95610834/149084198-b5224825-8a6e-4091-87e5-b80d2a02d89d.png)

**Expected behavior**
Only those services that need to be restarted should be offered to the user.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Edge Chromium
 - Version: 97.0.1072.55

**Comment by JoeMudr** [comment]
This is still an issue in 12.1.
Also on a clean ZEN installation radiusd-acct was enabled and continuously tried to restart like in https://github.com/inverse-inc/packetfence/issues/5232

**Comment by nqb** [comment]
This request looks legit.

However, I look on a ZEN 12.1 @JoeMudr and `radiusd-acct` is disabled after configurator.

**Comment by JoeMudr** [comment]
@nqb you are right. I just installed another PF instance where this service is indeed deactivated. Weird. So you can disregard my message above. But still: If this service doesn't serve any purpose anymore, why not remove it entirely?
